### PR TITLE
DOCS-1340: Update Split Payments documentation

### PR DESCRIPTION
**Pull request by Kris-MultiSafepay**
Do the changes meet MultiSafepay Docs' Definition of Done?
- Style guide, tone of voice, and naming conventions
- Tested
- Reviewed on mobile, tablet and desktop screens
- Notified stakeholders
- Pass GitHub Actions

### CODE REVIEW

File: content/payments/features/split-payments/_index.md [hunk 1]
```diff
@@ -15,17 +15,26 @@ aliases:
...
 
+Split Payments lets you divide an incoming transaction amount between two or more [partner/primary](/account/account-types/) or [affiliate](/account/account-types/) accounts, e.g. as a fee for using your platform.
 
-You can split payments by percentage, a fixed amount, or by both. 
+You can split payments by percentage, a fixed amount, or by both.
 
-See API reference – [Split Payments orders](/api/#split-payments-orders).
+## Integration
+
+Split Payments are made by [appending an object specifying the split](/api/#split-payments-orders) to the payload of the API request to create an order.
```
Comment (FrancescaNCook) [inline content/payments/features/split-payments/_index.md:25]:
```suggestion
To split a payment, [add an object specifying the split](/api/#split-payments-orders) to the payload of a `POST /orders` request.

```

File: content/payments/features/split-payments/_index.md [hunk 2]
```diff
@@ -15,17 +15,26 @@ aliases:
...
+
+Split Payments can be created by:
+
+- Affiliated accounts, e.g. for franchises splitting 10% of all incoming transactions to the linked primary account
+- Partner/primary accounts, e.g. for mixed-baskets where customers buy products from multiple affiliated merchants in a single transaction on a marketplace. 
+
+Incoming transaction amounts can be split between all account types and affiliations.
```
Comment (FrancescaNCook) [inline content/payments/features/split-payments/_index.md:32]:
```suggestion
```

Comment (FrancescaNCook) [reply content/payments/features/split-payments/_index.md:32]: Move this up to the introduction.

File: content/payments/features/split-payments/_index.md [hunk 3]
```diff
@@ -15,17 +15,26 @@ aliases:
     - /tools/split-payments/how-do-i-refund-split-payment-orders
     - /payments/features/split-payments/
 ---
-Split Payments lets you divide an incoming transaction amount between two or more [partner](/account/account-types/) or [affiliate](/account/account-types/) accounts, e.g. as a fee for using your platform.
 
+Split Payments lets you divide an incoming transaction amount between two or more [partner/primary](/account/account-types/) or [affiliate](/account/account-types/) accounts, e.g. as a fee for using your platform.
```
Comment (FrancescaNCook) [inline content/payments/features/split-payments/_index.md:19]:
```suggestion
Split Payments lets you divide an incoming transaction amount between two or more MultiSafepay accounts. This feature is supported for all [account types](/account/account-types/) and affiliations, e.g.:

- Partner/primary accounts, e.g. for mixed baskets where customers buy products from multiple affiliated merchants in a single transaction on your marketplace
- Affiliated accounts, e.g. for franchises splitting 10% of all incoming transactions to the linked primary account
```

File: content/payments/features/split-payments/_index.md [hunk 4]
```diff
@@ -15,17 +15,26 @@ aliases:
...
+Split Payments can be created by:
+
+- Affiliated accounts, e.g. for franchises splitting 10% of all incoming transactions to the linked primary account
+- Partner/primary accounts, e.g. for mixed-baskets where customers buy products from multiple affiliated merchants in a single transaction on a marketplace. 
+
+Incoming transaction amounts can be split between all account types and affiliations.
 
 ## Refunds
-You can only refund split payments (in full or in part) from the account that originally received the funds and then split them to other accounts. 
+You can only refund Split Payments (in full or in part) from the account that created the Split Payment.
```
Comment (FrancescaNCook) [inline content/payments/features/split-payments/_index.md:35]:
```suggestion
You can process full and partial refunds, but only from the account that created the split payment. 

```

File: content/payments/features/split-payments/_index.md [hunk 5]
```diff
@@ -15,17 +15,26 @@ aliases:
...
+- Partner/primary accounts, e.g. for mixed-baskets where customers buy products from multiple affiliated merchants in a single transaction on a marketplace. 
+
+Incoming transaction amounts can be split between all account types and affiliations.
 
 ## Refunds
-You can only refund split payments (in full or in part) from the account that originally received the funds and then split them to other accounts. 
+You can only refund Split Payments (in full or in part) from the account that created the Split Payment. 
 
-To refund a split payment, follow these steps:
+To refund a Split Payment, follow these steps:
```
Comment (FrancescaNCook) [inline content/payments/features/split-payments/_index.md:37]:
```suggestion
To refund a split payment, follow these steps:

```

Comment (FrancescaNCook) [reply content/payments/features/split-payments/_index.md:37]: My take: Split Payments is the name of the feature. Each instance is simply a split payment. What do you think?

**Review by FrancescaNCook — CHANGES_REQUESTED** [review]
Nice! Love the use case examples! A few tweaks...

File: content/payments/features/split-payments/_index.md (hunk 5 above)
Comment (Kris-MultiSafepay) [reply content/payments/features/split-payments/_index.md:37]: Yeah, I like that. In some contexts it's a bit tricky to distinguish between feature or instance of the feature, like in [this line](https://github.com/MultiSafepay/docs/pull/2057/files#diff-a63a92a3e1d5e881c174e66262714ceda807b73bb493550eaaaa760b13865074R35), but I'll try my best ;)

**Pull request merged**